app/posts/manage-teacher-training-applications/2020-12-18-search-by-candidate-name-or-reference.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Search by candidate name or reference
+description: Let users search the applications list by candidate name or reference
+date: 2020-12-18
+---
+
+We updated the search to allow users to find applications by reference in addition to candidate name.
+
+The search works independently from the filters, allowing users to filter or search in whatever way suits their needs.
+
+This change included:
+
+- moving the search form to above the applications list to accept terms that are compared to a candidate’s full name or reference
+- removing the candidate name search from the filters side bar
+- adding the reference number to each application listing
+- retaining the search term in the search field
+- providing a way to clear the search term once a search has been carried out, allowing users to get back to the original list state
+- updating the text and position of the clear filter action, which also resolved one of the issues highlighted in the recent accessibility review
+
+Other considerations included:
+
+- highlighting the search term in the applications that match
+- introducing a sorting mechanism (eg by relevance, alphabetical ascending/descending, etc) to allow users to sort the list beyond the default priority
+
+Neither considerations were implemented in this iteration as we could not establish a need for them.
+
+## Why we are doing this
+
+Providers have highlighted that when a candidate contacts them, they sometimes refer to their reference number. Being able to search by reference number as well as the candidate's name helps providers search more easily and accurately.
```

Comment:
I think this should be the first sentence of the post. WHY should come first.

Reply:
What does the word ‘highlight’ mean? Did they say they wanted this in research? Is it something they already do with UCAS?

Why does the reference help specifically in comparison to searching by name which we currently allow?

Reply:
>What does the word ‘highlight’ mean?

These are the words from the researchers. Happy to reframe if confusing.

>Why does the reference help specifically in comparison to searching by name which we currently allow?

The second sentence addresses this. Searching by reference is more accurate compared to searching by name since many candidates can have the same name, but a single candidate has a single reference number. I'll update it.